microsoft/testsuites/network/networksettings.py

```diff
@@ -91,6 +91,9 @@ class NetworkSettings(TestSuite):
 
         """,
         priority=1,
+        requirement=simple_requirement(
+            unsupported_os=[BSD],
```

Comment:
Please exclude Windows also. Windows is by default excluded. But after the change, it's included now.

Reply:
Added Windows to the unsupported OS list.